install.sh

```diff
@@ -257,6 +259,11 @@ function get_installation_options() {
 
   VOLUMES_MOUNT_PARAM=$([ -z $VOLUME ] && echo '' || echo "-v $VOLUME@server:*")
 
+  if [ "$FIX_DNS" == "true" ]
+  then
+    VOLUMES_MOUNT_PARAM="$VOLUMES_MOUNT_PARAM -v /etc/resolv.conf:/etc/resolv.conf@server:*"
+  fi
```

Comment:
Are you sure it doesn't break anything?
Did you check it on a clean machine?

Reply:
I've checked it on my machine